WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -365,14 +331,13 @@ private extension ProductsViewController {
         let cancelAction = UIAlertAction(title: Localization.cancel, style: .cancel)
 
         actionSheet.addAction(updateStatus)
-        if !viewModel.onlyVariableProductsSelected {
+        if !viewModel.onlyNonSimpleProductsSelected {
```

Comment:
I think this is too restrictive. As I understand it, we should be able to bulk update prices for both Simple and External/Affiliate products, but not Variable and Grouped products.

We should either:
1. Allow editing the price for both Simple and External/Affiliate products.
2. Disallow editing the price for Variable and Grouped products.

I'd lean toward the second option (identify and restrict the products that _don't_ support price editing) so that bulk editing prices can also be an option for non-core product types that support it. Otherwise, any custom product types enabled by extensions will also be excluded from bulk editing their prices.

Reply:
For point 3 we should also add all possible incompatible custom types in the future. For example, in beta test "booking" was ignoring bulk editing of a price.

But now I agree that silently ignoring some incompatible options is better than cutting off all custom types.

Reply:
Updated compatibility filter logic in a36fadd. Now it will filter only variable, grouped and bookings.

Reply:
This is great, and we can always add to the list of filtered products if we discover more custom types we want to exclude. Thanks for the documentation, as well!